lib/membrane/kino/input/input_bin_source.ex

```diff
@@ -84,7 +85,20 @@ defmodule Membrane.Kino.Input.Bin.Source do
           |> bin_output(pad)
       end
 
-    state = put_in(state.tracks[pad], Track)
     {[spec: spec], state}
   end
+
+  defp assert_pad_count(name, ctx) do
```

Comment:
```suggestion
  defp assert_pad_number!(name, ctx) do
```
Functions that check the assertion and raise if the condition in it isn't met should end with `!`